packages/smooth_app/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -8,7 +8,7 @@
 
 /* Begin PBXBuildFile section */
```

Comment:
What happened in this file

Reply:
I haven't the slightest idea. Do you mean that I should not include it in the PR?

Reply:
It's probably something flutter changes so it shouldn't be a problem.

Some native code paths from packages we either added or removed got changed and these larger code blocks moved and changed a few values but I can't say what the things are really for.

@teolemon maybe you wan't to have a look at this

Reply:
From my point of view it's just a piece of junk like `pubspec.lock`: nobody edits it but it lives its own life and sometimes we're supposed (or not) to commit it.

Reply:
It seems like autogenerated code, it does not seem to break any test, and it might actually be useful, so why not ?

Reply:
BUT: can you explain me what's going on with the pubspec.yaml (eg going back for version nbrs of libs) ?

Reply:
cc @monsieurtanuki

Reply:
I don't think we need to remove them and unless we have some concerns and let dart do its stuff itself